### PR TITLE
Differentiate between version.plist not existing and being unreadable

### PR DESCRIPTION
**Pull request by thefuzzyjew**
Added some logic to clarify if version.plist doesn't exist, or if it simply cannot be read. When running make_munki_mpkg.sh in a PPPC protected directory the output would state the file did not exist. However, the file did exist, but was unable to be read because of PPPC.

Any feedback appreciated.

If accepted I would appreciate the "hacktoberfest-accepted" label for [Hacktoberfest](https://hacktoberfest.digitalocean.com)

### CODE REVIEW

File: code/tools/make_munki_mpkg.sh [hunk 1]
```diff
@@ -153,12 +153,17 @@ if [[ "$CONFPKG" == "YES" ]] ; then
...
-if [ "$?" != "0" ]; then
-    echo "$MUNKIROOT/code/client/munkilib/version is missing!" 1>&2
-    echo "Perhaps $MUNKIROOT does not contain the munki source?" 1>&2
+VERSIONFILE="$MUNKIROOT/code/client/munkilib/version"
+# Check to see if file exists
+if [ -f "$VERSIONFILE.plist" ]; then
+    # Get the munki version
+    MUNKIVERS=$(defaults read "$VERSIONFILE" CFBundleShortVersionString)
+    if [ "$?" != "0" ]; then
+        echo "$VERSIONFILE can not be read" 1>&2
```
Comment (gregneagle) [inline code/tools/make_munki_mpkg.sh:162]: probably should be `echo "${VERSIONFILE}.plist can not be read" 1>&2`

File: code/tools/make_munki_mpkg.sh [hunk 2]
```diff
@@ -153,12 +153,17 @@ if [[ "$CONFPKG" == "YES" ]] ; then
...
+        exit 1
+    fi
+else
+    echo "$VERSIONFILE Missing" 1>&2
```
Comment (gregneagle) [inline code/tools/make_munki_mpkg.sh:166]: probably should be `echo "${VERSIONFILE}.plist missing" 1>&2`

**Review by gregneagle — CHANGES_REQUESTED** [review]
Two small adjustments requested. I cannot merge until there is a signed CLA on file.

File: code/tools/make_munki_mpkg.sh (hunk 1 above)
Comment (thefuzzyjew) [reply code/tools/make_munki_mpkg.sh]: Fixed in da9c2ac1852300274b81d65fb5805057fe645e90

File: code/tools/make_munki_mpkg.sh (hunk 2 above)
Comment (thefuzzyjew) [reply code/tools/make_munki_mpkg.sh]: Fixed in Fixed in da9c2ac1852300274b81d65fb5805057fe645e90

**Comment by thefuzzyjew** [comment]
CLA should be on file now.

**Pull request merged**